lib/node_modules/@stdlib/blas/ext/last-index-of/README.md

```diff
@@ -0,0 +1,260 @@
+-   **fromIndex**: index from which to begin searching (_optional_). May be either a scalar value or an [ndarray][@stdlib/ndarray/ctor] having an integer index or "generic" [data type][@stdlib/ndarray/dtypes]. If provided an [ndarray][@stdlib/ndarray/ctor], the value must have a shape which is [broadcast-compatible][@stdlib/ndarray/base/broadcast-shapes] with the non-reduced dimensions of the input [ndarray][@stdlib/ndarray/ctor]. For example, given the input shape `[2, 3, 4]` and `options.dim=0`, a provided [ndarray][@stdlib/ndarray/ctor] must have a shape which is [broadcast-compatible][@stdlib/ndarray/base/broadcast-shapes] with the shape `[3, 4]`. If provided a negative integer, the index at which to begin searching along a dimension is determined by counting backward from the last element (where `-1` refers to the last element). Default: `-1`.
+-   **options**: function options (_optional_).
+
+The function accepts the following options:
+
+-   **dtype**: output ndarray [data type][@stdlib/ndarray/dtypes]. Must be an integer index or generic [data type][@stdlib/ndarray/dtypes].
+-   **dim**: dimension over which to perform operation. If provided a negative integer, the dimension along which to perform the operation is determined by counting backward from the last dimension (where `-1` refers to the last dimension). Default: `-1`.
+-   **keepdims**: boolean indicating whether the reduced dimensions should be included in the returned [ndarray][@stdlib/ndarray/ctor] as singleton dimensions. Default: `false`.
+
+If the function is unable to find a search element along an [ndarray][@stdlib/ndarray/ctor], the corresponding element in the returned [ndarray][@stdlib/ndarray/ctor] is `-1`.
+
+```javascript
+var array = require( '@stdlib/ndarray/array' );
+
+// Create an input ndarray:
+var x = array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+// returns <ndarray>
+
+// Perform operation:
+var out = lastIndexOf( x, 10.0 );
+// returns <ndarray>
+
+var idx = out.get();
+// returns -1
+```
+
+By default, the function begins searching from the last element along the reduction dimension. To begin searching from a different index, provide a `fromIndex` argument.
+
+```javascript
+var array = require( '@stdlib/ndarray/array' );
+
+// Create an input ndarray:
+var x = array( [ 1.0, 2.0, 3.0, 4.0, 2.0, 6.0 ] );
+// returns <ndarray>
+
+// Perform operation:
+var out = lastIndexOf( x, 2.0, 3 );
+// returns <ndarray>
+
+var idx = out.get();
+// returns 2
+```
+
+By default, the function performs the operation over elements in the last dimension. To perform the operation over a different dimension, provide a `dim` option.
+
+```javascript
+var ndarray2array = require( '@stdlib/ndarray/to-array' );
+var array = require( '@stdlib/ndarray/array' );
+
+var x = array( [ [ -3.0, 2.0 ], [ -3.0, 4.0 ] ] );
+
+var out = lastIndexOf( x, -3.0, {
+    'dim': 0
+});
+// returns <ndarray>
+
+var idx = ndarray2array( out );
+// returns [ 0, -1 ]
```

Comment:
```suggestion
// returns [ 1, -1 ]
```

Shouldn't this be `[ 1, -1]`, if the matrix is

```
[
  [ -3.0, 2.0 ],
  [ -3.0, 4.0 ]
]
```

where `dim=0`, this means down the columns. The last matching element is element `x(1,0)`, not `x(0,0)`.